controller.rst

```diff
@@ -606,13 +606,16 @@ The ``file()`` helper provides some arguments to configure its behavior::
 Final Thoughts
 --------------
 
-Whenever you create a page, you'll ultimately need to write some code that
-contains the logic for that page. In Symfony, this is called a controller,
-and it's a PHP function where you can do anything in order to return the
-final ``Response`` object that will be returned to the user.
+In Symfony, a controller is usualll a class method which is used to accept requests,
+and return a ``Response`` object. Symfony provides an  ``AbstractController`` which
+can be used to extend the controller class, and alllows access to some frequently used utilties 
```

Comment:
`utilties` -> `utilities`

Reply:
This typo identified below has been addressed.
```suggestion
can be used to extend the controller class, and alllows access to some frequently used utilties 
```